io.github.rinigus.OSMScoutServer.json

```diff
@@ -0,0 +1,201 @@
+{
+    "app-id": "io.github.rinigus.OSMScoutServer",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "osmscout-server",
+    "finish-args": [
+        "--filesystem=host",
+        "--device=dri",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--talk-name=org.freedesktop.DBus"
```

Comment:
Overly broad.

Reply:
X11 & Wayland - it has GUI hence the permissions

Network - for serving maps and downloading new maps from the distribution servers

Access to host FS - covers the case where users want to store the maps somewhere on HDD, not necessarily home

dbus - to follow clients using map matching.

Reply:
I meant the dbus one specifically. Only add talk permissions for the specific names required not all of dbus.

Reply:
I'll drop DBus. At this stage, it will not work anyway. I am using `org.osm.scout.server1` on session dbus for interface and it seems to be incompatible with flatpak. Or can I use that name from this package? If not, its not a major deal. Its better to get it publish without it for now.

Reply:
If the app-id of the package is `org.osm.scout` or `org.osm.scout.server1` then you get ownership permissions to that automatically.

Reply:
(`server1` is a bit odd of a name ofc)

Reply:
I don't own scout.osm.org as a domain name - hence the name with github. As for server1, it follows API versioning recommendations at https://dbus.freedesktop.org/doc/dbus-api-design.html

Reply:
Thanks! Then I will add it. Now, to make DBus support complete - I am following applications on 

"/org/freedesktop/DBus", "org.freedesktop.DBus", "NameOwnerChanged"

This is done to keep an eye on the clients that registered map matching service - namely when they are closed. As soon as a client is closed, the server is removing map matching data requested by the client. How can I specify that I am following only that signal? Or is 

        "--talk-name=org.freedesktop.DBus"

the only way?

Reply:
That is just disabling the sandbox entirely...

I'm not familiar with an ideal DBus pattern off the top of my head though.

Reply:
Not sure it disables sandbox entirely, but I am not an expert on DBus. It still can initiate talk only with the specified services, not with some `a.b.c`

Reply:
Ok not entirely but you can do things through `org.freedesktop.DBus` you shouldn't be able to.